core/save-git.c

```diff
@@ -861,6 +861,10 @@ static void save_settings(git_repository *repo, struct dir *tree)
 	call_for_each_dc(&b, save_one_device, false);
 	cond_put_format(autogroup, &b, "autogroup\n");
 	save_units(&b);
+	if (prefs.tankbar)
+		put_string(&b, "prefs TANKBAR\n");
+	if (prefs.dcceiling)
+		put_string(&b, "prefs DCCEILING\n");
 
```

Comment:
haven't tried the code, but this makes me think that since the main prefs don't get set in the code above, if I modify things in Subsurface-mobile and then save them again to git, the tankbar / dcceiling setting will get lost, correct?